sw/host/opentitantool/src/command/lc.rs

```diff
@@ -54,8 +56,87 @@ impl CommandDispatch for LcStateRead {
     }
 }
 
+#[derive(Debug, Args)]
+/// Initiates a device transition from Raw to TestUnlocked0.
+pub struct RawUnlock {
+    /// The raw unlock token hexstring.
+    #[arg(long, default_value = "0x00000000_00000000_00000000_00000000")]
```

Comment:
Does the all-zero token have a special meaning (e.g. is it the default in a fake-keys build)? If not, maybe we shouldn't provide a default so this argument is mandatory and Clap will fail if you don't provide one

Reply:
fixed.